### ISSUE TITLE
Medic talents do not increase acid/poison damage + other problems

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#10920]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

**What a Stench** and **Macrodosing**:

1. Do not increase Acid damage as stated
2. Do not Affect items that *are* poisons
3. Affect "non-poisons" such as opiates

![image](https://user-images.githubusercontent.com/73229309/212177635-3816f0c9-e36a-4682-bcd9-7b9aeadc4d45.png)
![image](https://user-images.githubusercontent.com/73229309/212177741-51c9a0bb-ce14-45f1-b9cd-4df3b8d3d88a.png)

The xml for both talent uses:
`<CharacterAbilityGiveStat stattype="PoisonMultiplier" value="0.1"/>`

As seen here :https://github.com/Regalis11/Barotrauma/blob/a4cee74f8bdf9ed52f639ff3950ff507155ff5cd/Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs#L2168

PoisonMultiplier multiplies the value of AFFLICTIONS of the poison type

1. Acid burn is not in fact a poison,  poisonmultiplier doesn't increase acid damage

2. Items that could be considered poisons, like Mudraptor Bane or even Husk eggs inflict afflictions that are are not poisons (Organ damage) and do not benefit from the talent

3. On the other hand, afflictions that are arguably not poisons such as Drunk, Radiation  and Opiate Withdrawal/Overdose, are all poisons. This makes Morphine/Fentanyl actively worse with these two talents

------

# Potential fix:

**Make PoisonMultiplier work based off item tags, not afflictions, just like MedicalItemApplyingMultiplier does, replace "medical" by "poison", and add the "poison" tag to all poison items (including raptor bane) + add the missing xml below to boost acid**
https://github.com/Regalis11/Barotrauma/blob/a4cee74f8bdf9ed52f639ff3950ff507155ff5cd/Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs#L2141

Acid is a little different considering acid emitters will be the ones inflicting acid burn, so it will only boost the syringe and explosion damage of grenades.

Final thoughts about acid items in general:  https://github.com/Regalis11/Barotrauma/discussions/10612
110% multiplier for poison does feel weak and could be at least be 120% or more.

```xml
<!--  boosts acid burn but only on syringe/explosions -->
    <AbilityGroupEffect abilityeffecttype="OnAddDamageAffliction">
      <Abilities>
        <CharacterAbilityModifyAffliction afflictionidentifiers="acidburn" addedmultiplier="0.1" />
      </Abilities>
    </AbilityGroupEffect>
```

### Version

0.20.16.1

**Comment by itchyOwl** [comment]
Addressed in Regalis11/Barotrauma-development#3883. 

Your approach was a good start, but I had to adjust it a bit, not least because we've just overhauled the poisons. Anyway, good that you noticed the issue!

**Comment by JeroonBosch** [comment]
Tested (multiple times) - seems in a good spot now
